apps/builder/app/shared/webstudio-data-migrator.ts

```diff
@@ -24,7 +24,7 @@ export const migrateWebstudioDataMutable = (data: WebstudioData) => {
     const property = styleDecl.property as string;
 
     // expand overflow shorthand into overflow-x and overflow-y longhands
-    if (property === "overflow") {
+    if (property === "overflow" || property === "transition") {
```

Comment:
I just have one doubt in this process of data migration.  This runs everytime when the app is loaded. So, instead of just the old data structure. This will migrate all the `unparsed` values too in the app if the users add them now. 
PS: The values for `transition` becomes unparsed now when added from advanced tab. Should we add a check to migrate only if the old value is `layers` type for transitions proeprty.

Reply:
This will migrate only if these property names are appeared. No patches sent if these properties are already longhands.

Reply:
Ah, right. We need to remove transition from mdn data

Reply:
Done, dropped transition shorthand from property aggregator too 👍